pkg/report/report.go

```diff
@@ -216,6 +217,15 @@ func latestEnd(start time.Time, testcases []*Testcase) time.Time {
 // Report prints a report for TestSuites to the directory.  ftype == json | xml
 func (ts *Testsuites) Report(dir, name string, ftype Type) error {
 	ts.Close()
+
+	// Create the folder to save the report if it doesn't exist
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
+		dirCreationErr := os.MkdirAll(dir, 0755)
```

Comment:
it would be great if we could shorten the name `dirCreationErr` to much shorter.   `creationErr` is better but we could also reuse the `err` here... if is throw away at this point anyway.

Reply:
Sure!